dbt_copilot_helper/commands/conduit.py

```diff
@@ -185,19 +202,52 @@ def add_stack_delete_policy_to_task_role(app: Application, env: str, addon_name:
             )
 
 
+def update_conduit_stack_resources(app: Application, env: str, addon_name: str, task_name: str):
+    session = app.environments[env].session
+    cloudformation_client = session.client("cloudformation")
+
+    conduit_stack_name = f"task-{task_name}"
+    template = cloudformation_client.get_template(StackName=conduit_stack_name)
+    template_yml = load_yaml(template["TemplateBody"])
+    template_yml["Resources"]["LogGroup"]["DeletionPolicy"] = "Retain"
+    template_yml["Resources"]["TaskNameParameter"] = load_yaml(
+        f"""
+        Type: AWS::SSM::Parameter
+        Properties:
+          Name: /copilot/{app.name}/{env}/secrets/{addon_name}_CONDUIT_TASK_NAME
```

Comment:
Maybe separating these parameters from the `secrets` as they are meant for services. Something like `/copilot/{app.name}/{env}/conduits/{addon_name}_CONDUIT_TASK_NAME` might be better.